src/main/java/teammates/common/util/Const.java

```diff
@@ -170,6 +170,8 @@ private Const() {
         public static final String QUERY_LOGS_VERSION = "version";
         public static final String QUERY_LOGS_EXTRA_FILTERS = "extrafilters";
         public static final String QUERY_LOGS_ORDER = "order";
+
+        public static final String LIMIT = "limit";
```

Comment:
I feel like this name can get more descriptive, say `DELETE_STUDENTS_BATCH_LIMIT`

Reply:
That was what I did initially, but I reasoned that there might be chances we use the limit param for other purposes/endpoints in the future, limit feels more like a general parameter than one that is specific to our delete all endpoint.